checks/collector.py

```diff
@@ -368,7 +371,19 @@ def run(self, checksd=None, start_event=True):
             payload['metrics'].extend(self._agent_metrics.check(payload, self.agentConfig,
                 collect_duration, self.emit_duration))
 
-
+        # If required by the user, let's create the app:xxx host tags
+        if self.agentConfig['create_app_tags']:
```

Comment:
It's not probably not the right location to send that. We shouldn't run this code at every iteration. It should probably be in this block:
https://github.com/DataDog/dd-agent/blob/etienne/per-app-tagging/checks/collector.py#L461